specs/_template/03_external_communication.rst

```diff
@@ -0,0 +1,87 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+=========================
+External Communication
+=========================
+
+.. contents:: Table of Contents
+
+1. Service A
+--------------------
+
+**Protocol:** JSON-RPC via REST
+
+- `Put your link to documentation here`_
+- `More links`_
+
+1.1. Operation Name
```

Comment:
I would prefer if we do not copy documentation content into specs. E.g. if we should use a relay function enter a link to the readthedocs or swagger and state the function name.

If the required endpoint doesn't exist, we should state that an endpoint needs to be defined. This can be done while working on the specs but should result in an update of the readthedocs and not only in the specs.